values.tf

```diff
@@ -129,17 +156,17 @@ locals {
     }
   })
```

Comment:
with new vector versions there are multiple configuration breaking changes for elasticsearch sink, we should update the default config to a functional version:

```
  helm_values_sink_opensearch = yamlencode({
    "customConfig" : {
      "sinks" : {
        "elasticsearch_kubernetes_containers" : {
          "type" : "elasticsearch",
          "inputs" : ["kubernetes_containers"],
          "endpoint" : var.opensearch_endpoint,
          "mode" : "data_stream",
          "bulk" : {
            "action" : "create"
          },
          "data_stream" : {
            "type" : "logs",
            "dataset" : "kubernetes",
            "namespace" : "pods"
          },
          "compression" : "gzip",
          "auth" : {
            "strategy" : "aws",
          },
          "aws": {
            "region" : data.aws_region.current.name
          }
        },
        "elasticsearch_journal" : {
          "type" : "elasticsearch",
          "inputs" : ["journal"],
          "endpoint" : var.opensearch_endpoint,
          "mode" : "data_stream",
          "bulk" : {
            "action" : "create"
          },
          "data_stream" : {
            "type" : "logs",
            "dataset" : "journal",
            "namespace" : "hosts"
          },
          "compression" : "gzip",
          "auth" : {
            "strategy" : "aws",
          },
          "aws": {
            "region" : data.aws_region.current.name
          }
        }
      }
    }
  })
  ```
  
  Note: aws.region is now required when using aws auth strategy, bulk syntax was changed